couchdb/couchdb.rb

```diff
@@ -160,7 +160,7 @@ def monitor_couchdb(couchdb_servers, group_name)
       begin
         uri = URI.parse("#{rhost['url']}/_stats?range=60")
 
-        if rhost['user'] and rhost['password']
+        if !rhost['user'].empty? and !rhost['password'].empty?
```

Comment:
Guys, there’s really something wrong with your github settings.
I’m getting emails about code changes and I can see your code even without being logged in to github.

From: Gagandeep singh [mailto:notifications@github.com]
Sent: Thursday, May 26, 2016 4:07 PM
To: CopperEgg/metrics-plugins
Subject: Re: [CopperEgg/metrics-plugins] CE-2009 : Fixed units for all metrics, Fixed case when CouchDB has no username and password (#16)


In couchdb/couchdb.rb<https://github.com/CopperEgg/metrics-plugins/pull/16#discussion_r64741220>:

> @@ -160,7 +160,7 @@ def monitor_couchdb(couchdb_servers, group_name)

>        begin

>          uri = URI.parse("#{rhost['url']}/_stats?range=60")

>

> -        if rhost['user'] and rhost['password']

> +        if !rhost['user'].empty? and !rhost['password'].empty?

replaced with unless

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub<https://github.com/CopperEgg/metrics-plugins/pull/16/files/3f3811c7d364eb4ecef1a3a80af92fe719ccda9d#r64741220>